setup.cfg

```diff
@@ -1,8 +1,39 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
-license_file = LICENSE
```

Comment:
This option has been deprecated. It's been replaced by `license_files`, but the `LICENSE` file will be automatically included in the distribution so it's not necessary here. (azavea/django-amazon-ses#54)